### ISSUE TITLE
Does Firefox 73.0.1 need a new css?

### ISSUE THREAD

**Issue by pystol777** [trogau/firefoxSettings#4]
Hi, are you still updating this css? I've resisted updating to 73 but accidentally pressed a key which started a Firefox refresh which automatically updated me - now my tabs are on top again. Grrrrrr. :) Hope you can help. Cheers

**Comment by trogau** [comment]
Yes! The current version is what I'm using in 73.0.1 and it seems to be
working OK for me.

-- 
dave

**Comment by unitb-ore** [comment]
The CSS for FF 72 still works for me in FF 73.
But on another PC it didn't work. Solution: The menu bar has to be enabled, then it works, otherwise not (tested with FF 72).
